python/lsst/base/threads.cc

```diff
@@ -30,11 +30,12 @@ namespace lsst {
 namespace base {
 
 PYBIND11_MODULE(threads, mod) {
-    mod.def("haveThreads", &lsst::base::haveThreads);
-    mod.def("setNumThreads", &lsst::base::setNumThreads);
-    mod.def("getNumThreads", &lsst::base::getNumThreads);
-    mod.def("disableImplicitThreading", &lsst::base::disableImplicitThreading);
+    mod.def("haveThreads", &lsst::base::haveThreads, "Are threaded packages available?");
```

Comment:
I'm a bit worried by this; because this is not our usual practice, someone modifying the docstrings in the C++ include file wouldn't know to change the one here.  I'd prefer to leave these out until we have a more standard solution.

Reply:
Okay. It was annoying me that now the sphinx docs have no docstrings at all but will remove.